src/framework/ui/button/type.ts

```diff
@@ -43,6 +43,9 @@ export class ButtonAlignments {
   }
 
   private static typeOf(value: any): value is ButtonAlignment {
+    if (!value) {
```

Comment:
Fixed with `ButtonGroup` implementation. Can be omitted

Reply:
Done